web/modules/custom/bglobal_angular/components/info_sessions_list/template.html

```diff
@@ -0,0 +1,105 @@
+<!-- 
+  Reference modules/custom/bglobal_angular/components/voices_list/template.html
+ -->
+
+<!-- <div data-ng-if="view.count > 0 && $parent.parentProgram.abbreves === 'pdp'">
+  <div data-ng-repeat="session in sessions | limitTo:1">
+    <p class="blockquote">"Please join us for a free <a data-ng-href="{{ session.enroll }}" target="_blank">online information session</a>."</p>
+    <p class="caption">&mdash;Mariam Rahimi, Admissions Adviser</p>
+  </div>
+</div> -->
+<div class="events">
+  <h3 class="block-header">Info Sessions</h3>
+  <ul class="media-list" *ngIf="view.count > 0">
+    <li class="media" *ngFor="let session of sessions">
+      <div class="media-left">
+        <div class="date">
+          <span class="month">{{ session.jsDate | date:'MMM' }}</span> <span class="day">{{ session.jsDate | date:'dd' }}</span>
+        </div>
+      </div>
+      <div class="media-body">
+        <h4>
+          <a [href] = "session.path">{{session.title}}</a>
+        </h4>
+     </div>
+    </li>
+  </ul>
+  <div *ngIf="view.count <= 0"><h1>No Info Sessions to show!</h1></div>
+</div>
+
+<style>
```

Comment:
Is this temporary until the styleguide work is done? I'm hesitant to use inline CSS here. How will this be tested for accessibility and responsiveness?